src/core/Akka/Routing/ScatterGatherFirstCompleted.cs

```diff
@@ -78,14 +79,18 @@ public ScatterGatherFirstCompletedRoutees(Routee[] routees, TimeSpan within)
         /// <param name="sender">The actor sending the message.</param>
         public override void Send(object message, IActorRef sender)
         {
```

Comment:
I'm a bit iffy on the error handling here. What happens if one of these tasks throws?